android/src/ti/intercom/TitaniumIntercomModule.java

```diff
@@ -130,23 +138,23 @@ public void updateUser(KrollDict user) {
 		UserAttributes.Builder userAttributes = new UserAttributes.Builder();
 
 		if (id != null) {
-			userAttributes = userAttributes.withUserId(id);
-		}
+            userAttributes.withUserId(id);
+        }
```

Comment:
Fix indentation of all the attribute checks (3 occurences)

Reply:
Ahh, it's mixed space and tab based indentation. Should be fine now.